akka-stream-tests/src/test/scala/akka/stream/impl/fusing/IteratorInterpreterSpec.scala

```diff
@@ -8,21 +8,21 @@ import akka.util.ByteString
 import akka.stream.stage._
 import akka.stream.Supervision
 
-class IteratorInterpreterSpec extends AkkaSpec {
+class IteratorInterpreterSpec extends AkkaSpec with GraphInterpreterSpecKit {
```

Comment:
(As far as I know) I need it in order to call `.toGS` on the non-GraphStage-stages below.

... And I need to do that because I changed the `IteratorInterpreter` to take a `Seq[GraphStageWithMaterializedValue[FlowShape[_, _], Any]]` instead of a `Seq[PushPullStage[_, _]]`, because the only usage of `IteratorInterpreter` I could find was [with a `BodyPartParser`](https://github.com/akka/akka/blob/bd8fcc9d9aeafc2c49745e7519d13b3e0abc18e6/akka-http/src/main/scala/akka/http/scaladsl/unmarshalling/MultipartUnmarshallers.scala#L84).